wcfsetup/install/files/lib/acp/form/CaptchaQuestionAddForm.class.php

```diff
@@ -3,198 +3,104 @@
 namespace wcf\acp\form;
 
 use wcf\data\captcha\question\CaptchaQuestionAction;
-use wcf\data\captcha\question\CaptchaQuestionEditor;
-use wcf\form\AbstractForm;
-use wcf\system\exception\UserInputException;
-use wcf\system\language\I18nHandler;
+use wcf\form\AbstractFormBuilderForm;
+use wcf\system\form\builder\container\FormContainer;
+use wcf\system\form\builder\field\BooleanFormField;
+use wcf\system\form\builder\field\MultilineTextFormField;
+use wcf\system\form\builder\field\TextFormField;
+use wcf\system\form\builder\field\validation\FormFieldValidationError;
+use wcf\system\form\builder\field\validation\FormFieldValidator;
 use wcf\system\Regex;
-use wcf\system\request\LinkHandler;
-use wcf\system\WCF;
 use wcf\util\StringUtil;
 
 /**
  * Shows the form to create a new captcha question.
  *
- * @author  Matthias Schmidt
+ * @property    CaptchaQuestionAction   $objectAction
+ *
+ * @author  Matthias Schmidt, Florian Gail
```

Comment:
```suggestion
 * @author  Florian Gail, Matthias Schmidt
```
Alphabetic sort by last name.